src/query/ast/tests/it/parser.rs

```diff
@@ -420,7 +420,7 @@ fn test_statement_error() {
         r#"alter database system x rename to db"#,
         r#"create user 'test-e'@'localhost' identified bi 'password';"#,
         r#"drop usar if exists 'test-j'@'localhost';"#,
-        r#"alter user 'test-e'@'localhost' identifie by 'new-password';"#,
+        r#"alter user 'test-e'@'localhost' identifies by 'new-password';"#,
```

Comment:
Switching to `identifies` to avoid using a word not in the dictionary / a word that's recognized by the parser.